File: BikeServiceApplication/src/objects/Task.js
class Task{
    static task = {
        TaskId: 0,
        Header: "",
        Description: "",
        State: 0,
        TaskIdKey: "",
        AppId: 0,
        InitDate: ""
    }

    static statusList = [
        {   
            Value : 0,
            Label: 'Brak danych'
        },
        {   
            Value : 1,
            Label: 'Przyjęte'
        },
        {   
            Value : 2,
            Label: 'W trakcie'
        },
        {   
            Value : 3,
            Label: 'Gotowe'
        },
        {   
            Value : 4,
            Label: 'Odebrane'
        }
    ]

    
}

export default Task;

File: WebService/BikeWebService/Models/Task.cs
using BikeWebService.Interfaces;

namespace BikeWebService.Models
{
    public class Task : ITask
    {
        public int taskID { get; set; } = 0;
        public int appID { get; set; } = 0;
        public string header { get; set; } = "";
        public string description { get; set; } = "";
        public int state { get; set; } = 0;
        public string taskIDKey { get; set; } = "";
    }
}